src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,11 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(age -> age >= 30)
```

Comment:
```suggestion
                .filter(age -> age >= 30 && age <= 45)
```
검사조건간의 우선순위가 없는 케이스이기 때문에 하나의 filter 에서 처리해볼 수 도 있을 것 같아요~!